hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/parsing/PopulateReqHeader.java

```diff
@@ -37,8 +37,12 @@ public void populateReqHeader(Exchange exchange, @Headers Map<String, String> hm
 		String msgType = Util.getMsgType(v2Message);
 		hm.put(RECEIVING_APP, recApp);
 		hm.put(MESSAGE_TYPE, msgType);
+		hm.put(Exchange.HTTP_RESPONSE_CODE, "200");
```

Comment:
Jetty has an HttpStatus class that contains constants for all the HTTP status codes.

Reply:
Done